pkg/bot/slack_cloud.go

```diff
@@ -519,14 +519,33 @@ func (b *CloudSlack) send(ctx context.Context, event slackMessage, resp interact
 	var file *slack.File
 	var err error
 	if len(markdown) >= slackMaxMessageSize {
-		file, err = uploadFileToSlack(ctx, event.Channel, resp, b.client, event.ThreadTimeStamp)
-		if err != nil {
-			return err
-		}
-		resp = interactive.CoreMessage{
-			Message: api.Message{
-				PlaintextInputs: resp.PlaintextInputs,
-			},
+		if strings.Contains(resp.Description, "logs") {
```

Comment:
Actually its for the command of ```kubectl logs```

Reply:
No, maybe there is a misunderstanding. The primary motivation is to not send an attachment for the interactive blocks. Those blocks are the interactive dropdown elements e.g. namespace, name, verb . So, once you execute `kubectl` it will show you interactive elements, so that elements should be limited so there wont be an attachment.  More concrete example is, once you select namespace, it will return resources (e.g. pods), if those pods > 100 you will see an attachment in the slack UI, you need to prevent that.

Reply:
Ya I get the primary thing here so ill be needing to just add an condition over non-interactive instead of logs right